pytest_testrail/plugin.py

```diff
@@ -221,22 +254,24 @@ def pytest_sessionfinish(self, session, exitstatus):
 
     # plugin
 
-    def add_result(self, test_ids, status, comment='', duration=0):
+    def add_result(self, test_ids, status, comment='', duration=0, defects=None):
```

Comment:
`defects` should be before `duration`

Reply:
Dear allankp,
Thanks for the comment.
I've just changed the order.